File: DataStructures and Algorithms/recursion/recursion.py
# Recurion example
def eat_food(tummy_full):
    if tummy_full  == 1  == 100:
        return "Stop eating" #base case 
    else: 
        print(f"Bite taken. Fullness: {tummy_full}")
        return eat_food(tummy_full + 1)


print (eat_food(0))

File: DataStructures and Algorithms/recursion/recursion.ts
// Recursion


function eat_food(tummy_full: number): string{
   if (tummy_full == 10){
       return "tummy is full";
   }else{
     console.log(`Took a bit. Fullness: ${tummy_full}`);
       eat_food(tummy_full + 1)
   }

} 


console.log(eat_food(8))

